react-search-component/my-app/src/SearchableList.tsx

```diff
@@ -0,0 +1,20 @@
+type search = {
+  searchString: string;
+};
+
+export default function SearchableList({ searchString }: search) {
+  const ListItems = [
```

Comment:
The constraints asked that these be passed in rather than defined here. The reason for that is to make this component more reusable. I suggest you make that change. It should be simple.